packages/iris-grid/src/mousehandlers/IrisGridColumnSelectMouseHandler.ts

```diff
@@ -1,23 +1,31 @@
 /* eslint class-methods-use-this: "off" */
-import { GridMouseHandler } from '@deephaven/grid';
+import {
+  GridMouseHandler,
+  GridPoint,
+  EventHandlerResult,
+} from '@deephaven/grid';
+import type { Column } from '@deephaven/jsapi-shim';
+import type { IrisGrid } from '../IrisGrid';
```

Comment:
This makes an import cycle - I think ultimately the IrisGrid type should be separated? Or perhaps the type passed into the constructor is limited to the props/state this mouse handler needs? Thoughts?

Reply:
`import type` is completely stripped from the output, so it's not forbidden. But also not necessarily the best design.

Limiting to the props on the mouse handler might be better overall as it doesn't need an `IrisGrid` specifically. Some other component could have the same props. Although I'm not sure how TS will handle this when `IrisGrid` is actually TS (it's JS now). The import type might just be `JSX.Element` which isn't helpful